### ISSUE TITLE
Estimate Three-Phase Distribution Line Parameters With Physics-Informed Graphical Learning Method. (arXiv:2102.09023v1 [cs.LG])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#3851]
Estimate Three-Phase Distribution Line Parameters With Physics-Informed Graphical Learning Method. (arXiv:2102.09023v1 [cs.LG])<br>
https://ift.tt/3ubuZMT<br>
<p>Accurate estimates of network parameters are essential for modeling, monitoring, and control in power distribution systems. In this paper, we develop a physics-informed graphical learning algorithm to estimate network parameters of three-phase power distribution systems. Our proposed algorithm uses only readily available smart meter data to estimate the three-phase series resistance and reactance of the primary distribution line segments. We first develop a parametric physics-based model to replace the black-box deep neural networks in the conventional graphical neural network (GNN). Then we derive the gradient of the loss function with respect to the network parameters and use stochastic gradient descent (SGD) to estimate the physical parameters. Prior knowledge of network parameters is also considered to further improve the accuracy of estimation. Comprehensive numerical study results show that our proposed algorithm yields high accuracy and outperforms existing methods.</p>
<br>
<br>
via eess.SY updates on arXiv.org http://arxiv.org/<br>
